package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.0.8",
   "main": "tslint.json",
   "scripts": {
-    "postinstall": "./install-hooks"
+    "postinstall": "$(git rev-parse --show-toplevel)/node_modules/.bin/tsc ./rules/*.ts; ./install-hooks"
```

Comment:
let's name what's going on here (not positive on the syntax; `npm run` may not be necessary):
```
"scripts": {
  "postinstall": "npm run compilerules; ./install-hooks",
  "compilerules": "$(git rev-parse --show-toplevel)/node_modules/.bin/tsc ./rules/*.ts"
}
```